site/templates/footer.jet

```diff
@@ -1,9 +1,9 @@
 {{block footer()}}
-	<footer>
+	<footer role="contentinfo">
 		<a href="{{ routeToPath("/") }}" class="footer-brand">
 			{{i18n("nav_homepage")}}
 		</a>
-		<div class="footer-nav">
+		<nav class="footer-nav" role="navigation" aria-label="footer">
```

Comment:
[Looks to be redundant](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role).
```suggestion
		<nav class="footer-nav" aria-label="footer">
```